File: SWIFT/Cap06/c06ex01.swift
// c06ex01.swift

import Foundation

var MD = [Double](repeating: 0.0, count: 8)

var SOMA: Double = 0.0
var MEDIA: Double
var I: Int

I = 0
while (I <= 7) {
    print("Entre a média do aluno \(I + 1): ", terminator: "")
    if let inputMD = readLine(), let media = Double(inputMD) {
        MD[I] = media
        SOMA += MD[I] 
        I += 1
    } else {
        print("\nEntrada inválida. Tente novamente.\n")
    }
}

MEDIA = SOMA / 8.0

print("\nMédia geral: \(String(format: "%.2f", MEDIA))")

print()
print("Tecle <Enter> para encerrar... ", terminator: "")
_ = readLine()

File: KOTLIN/Cap06/c06ex01.kt
// c06ex01.kt

fun main() {
    var MD = Array(8) { 0.0f }

    var SOMA: Double = 0.0
    var MEDIA: Double

    for (I in 0..7) {
        print("Entre a media do aluno ${I + 1}: ")
        MD[I] = readLine()?.toFloatOrNull() ?: 0.0f
        SOMA += MD[I]
    }

    MEDIA = SOMA / 8

    println()
    println("Media geral: %.2f".format(MEDIA))

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
